Monika After Story/game/script-topics.rpy

```diff
@@ -17379,3 +17379,60 @@ label monika_quiet_time:
         m 2lksdlc "...I would appreciate if you could show me some kindness, though..."
         m 2dksdlc "..."
     return
+
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_hello_monika",
+            category=['life'],
+            prompt="Hello Monika!",
+            pool=True
+        )
+    )
+    
+label monika_hello_monika:
+
+    python:
+    
+        monika_hello_random_responses = ["Hi, {0}!".format(mas_get_player_nickname()),"Hello to you too, {0}!".format(player),"Hey, {0}! What's up?".format(player),"¡Hola {0}!".format(player),"Bonjour, {0}!".format(mas_get_player_nickname()),"Hi, {0}!".format(player),"Hey, {0}! How's it going?".format(mas_get_player_nickname()),"Hey hey!","Oh! {0}!".format(mas_get_player_nickname()),"Ah! {0}!".format(player)]
```

Comment:
I don't think it's really needed to use ```"{0}".format(...)``` things.
Like this 17398 line, you can simply write ```"Hi, [mas_get_player_nickname()]!"```.